lib/wprof/conf_wprof.rb

```diff
@@ -0,0 +1,25 @@
+module WProf
+  class Config
+    DEFAULTS_CONFIGS = {
+      db_runtime: true,
+      reporter_type: 'LOGGER',
+      csv_type: 'SPLIT',
+      async: false,
+      httparty_methods_to_trace: %w[get put delete post patch].freeze,
+      external_url: nil,
+      external_headers: nil,
+      custom_methods: nil,
+      file_path: Rails.root.to_s
+    }.freeze
+
+    class << self
+      def get_value(param)
+        unless Rails.application.nil?
```

Comment:
Is this validation necesary?

Reply:
Yes... Rails load gem data before defined his instance, Rails.application are not available in that moment. Without this validation, Rails don't run.